app/src/main/res/xml/pref_tidepool.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:validate="http://schemas.android.com/apk/res-auto">
 
     <PreferenceCategory
+        android:key="@string/tidepool"
```

Comment:
android:key shouldn't be "@string/key_tidepool" or "@string/key_tidepool_settings" ?